registry/txt.go

```diff
@@ -99,7 +99,9 @@ func (im *TXTRegistry) Records() ([]*endpoint.Endpoint, error) {
 	}
 
 	for _, ep := range endpoints {
-		ep.Labels = endpoint.NewLabels()
+		if nil == ep.Labels {
```

Comment:
Can you change it to ep.Labels == nil? I thinks its a bit more go idiomatic right?

Reply:
Can we also have a test for that?

Reply:
> Can you change it to ep.Labels == nil? I thinks its a bit more go idiomatic right?

I've flipped it around. Yeah it's probably more idiomatic on Go than in C, where I got used to writing rvalues on the left and lvalues on the right in comparison operations because then the compiler would flag an error if you wrote "=" instead of "==". Since assignments don't return a value in Go, it'll always flag an error and thus the order of the operators doesn't matter there.